Sources/SettingsFeature/Settings.swift

```diff
@@ -314,8 +315,7 @@ public let settingsReducer = Reducer<SettingsState, SettingsAction, SettingsEnvi
       }
 
     case .binding(\.sendDailyChallengeReminder):
-      struct CancelId: Hashable {}
```

Comment:
Unique tokens meant that rapidly changing both settings could lead to some UI hiccups as the updated settings are returned from the server.